packages/chat-client/src/client.ts

```diff
@@ -106,9 +106,18 @@ export class ChatClient extends IChatClient {
     }
   };
 
-  public resolve: IChatClient["resolve"] = async (params) => {
+  public resolveIdentity: IChatClient["resolveIdentity"] = async (params) => {
```

Comment:
Bit confused about this since I can't find the spec changes for these top-level client changes:
* Current docs still say `resolve`
* I see this PR but doesn't seem to touch this method: https://github.com/WalletConnect/walletconnect-docs/pull/433/files#diff-9e5c8391843ba9d7946a0b09dfe284640b612530e2f1fcff0b808c788ab78ab2

This is effectively breaking so we can go for `v1.x.x` once this goes to `main` I'm assuming right?

Reply:
Yes I'm going to add the top level `resolve` again once I'm done with everything, but this just helps keep everything clear (to me, at least) since we're resolving too different things now